aeron-tools/src/main/java/uk/co/real_logic/aeron/tools/perf_tools/AeronPing.java

```diff
@@ -0,0 +1,378 @@
+/*
```

Comment:
This version is more for the performance testing framework. I didn't want to clutter up the sample version (which I think most people would use). This version is run after builds and then the data is fed through gnuplot to generate our latency graphs per build. I will clean it up and change a couple of method names since I've started using gnuplot instead of having the tool generate scatterplots itself.